pkg/devcontainer/compose.go

```diff
@@ -794,3 +794,7 @@ func mappingToMap(mapping composetypes.MappingWithEquals) map[string]string {
 	}
 	return ret
 }
+
+func isDockerComposeConfig(config *config.DevContainerConfig) bool {
+	return len(config.DockerComposeFile) > 0
+}
```

Comment:
Mind also using this function [here](https://github.com/loft-sh/devpod/blob/main/pkg/devcontainer/run.go#L161)?

Reply:
Thanks @lizardruss . Good catch!! I have updated the code now to reuse the function there.